README.md

```diff
@@ -1,40 +1,33 @@
-- **Owner**:
-  [@sfoucher](https://github.com/sfoucher)
-  [@fmigneault](https://github.com/fmigneault)
-  [@ymoisan](https://github.com/ymoisan)
-
-[stac-ext-maturity]: https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity
-
-This document explains the Template Extension to the [SpatioTemporal Asset Catalog][stac-spec] (STAC) specification.
-This document explains the fields of the STAC Deep Learning Model (dlm) Extension to a STAC Item. 
-The main objective is to be able to build model collections that can be searched 
-and that contain enough information to be able to deploy an inference service.
-When Deep Learning models are trained using satellite imagery, it is important
-to track essential information if you want to make them searchable and reusable:
-1. Input data origin and specifications
-2. Model base transforms
-3. Model output and its semantic interpretation
-4. Runtime environment to be able to run the model
+- **Extension Maturity Classification:** Proposal
+- **Owner:**
+  - [@sfoucher](https://github.com/sfoucher)
+  - [@fmigneault](https://github.com/fmigneault)
+  - [@ymoisan](https://github.com/ymoisan)
+  - [@rbavery](https://github.com/rbavery)
+
+This document explains the Template Extension to the [SpatioTemporal Asset Catalog (STAC)](https://github.com/radiantearth/stac-spec) specification. This document explains the fields of the STAC Model Extension to a STAC Item. The main objective of the extension is two-fold: 1) to enable building model collections that can be searched alongside associated STAC datasets and 2) to record all necessary parameters, artifact locations, and high level processing steps to deploy an inference service. Specifically, this extension records the following info to make ML models searchable and reusable:
+1. Sensor band specifications
+2. The two fundamental transforms on model inputs: rescale and normalization
+3. Model output shape, data type, and its semantic interpretation
+4. An optional, flexible description of the runtime environment to be able to run the model
 5. Scientific references
 
-[stac-spec]: https://github.com/radiantearth/stac-spec
+Note: The spec is biased towards supervised ML models the produce classifications. However, fields that relate to supervised ML are optional and users can use the fields they need for different tasks.
 
-Check the original technical report
-[here](https://github.com/crim-ca/CCCOT03/raw/main/CCCOT03_Rapport%20Final_FINAL_EN.pdf) for more details.
+Check the original technical report for an earlier version of the Model Extension [here](https://github.com/crim-ca/CCCOT03/raw/main/CCCOT03_Rapport%20Final_FINAL_EN.pdf) for more details.
 
-![](https://i.imgur.com/cVAg5sA.png)
+![Image Description](https://i.imgur.com/cVAg5sA.png)
 
 - Examples:
-  - [Example with a UNet trained with thelper](examples/item.json)
+  - [Example with a ??? trained with torchgeo](examples/item.json) TODO update example
```

Comment:
thanks! I'd like to use torchgeo for the example since I think it is the most popular framework specific to EO for training deep learning models. They have pretrained models on EO datasets, Sentinel-2, Landsat, and they intend to host others.

I've been working with their 13 band pretrained Sentinel-2 Resnet model and was thinking of proposing this as the new example: https://torchgeo.readthedocs.io/en/stable/tutorials/trainers.html

Reply:
Yes, I agree to use the more popular torchgeo. Just need to update the example with the corresponding output. The Sentinel-2 Resnet is good. I've been using it also.

Reply:
huggingface is down now but I'll upload the Torchscript version of https://huggingface.co/torchgeo/resnet18_sentinel2_all_moco/ I've been working with that corresponds to the example.json

Reply:
coming back to this, I can upload this after remaking a notebook demo with the new schema and a new version of stac_model